pyodide.asm.js

```diff
@@ -10411,7 +10418,7 @@ var _createPyodideModule = (function () {
                     symName = symName.substring(5);
                     replace = true;
                 }
-                baseName = symName;
+                var baseName = symName;
```

Comment:
Do you remember the errors you were getting before this change. We should probably add a unit test to Pyodide for this code path in Node.

Reply:
This is the error that I got without this change:
```
ReferenceError: baseName is not defined
	at updateGOT (c:\GIT\s p\vscode-pyo\out\dist\pyodide.asm.js:6378:26)
	at relocateExports (c:\GIT\s p\vscode-pyo\out\dist\pyodide.asm.js:6448:13)
	at receiveInstance (c:\GIT\s p\vscode-pyo\out\dist\pyodide.asm.js:2870:27)
	at receiveInstantiatedSource (c:\GIT\s p\vscode-pyo\out\dist\pyodide.asm.js:2876:17)
```